lib/helpers.js

```diff
@@ -157,34 +211,110 @@ function buildMessage (prs, username) {
       ]
     }
 
+    if (branches.length > 0) {
+      // since the branches message can be so long that MS Teams won't even show
+      // the message, erase everything that was added from the PRs and create a
+      // new message
+      message = []
```

Comment:
This comment doesn't make a lot of sense to me. What happens to the erased message? Was it already posted?

Reply:
OK, now I'm seeing that the `buildMessage` is called twice with different params. My gut tells me that these should just be two separate methods, especially if we're just erasing this variable at this stage.